website/source/docs/builders/hetzner-cloud.html.md

```diff
@@ -0,0 +1,82 @@
+---
+description: |
+    The Hetzner Cloud Packer builder is able to create new images for use with the
+    Hetzner Cloud. The builder takes a source image, runs any provisioning
+    necessary on the image after launching it, then snapshots it into a reusable
+    image. This reusable image can then be used as the foundation of new servers
+    that are launched within the Hetzner Cloud.
+layout: docs
+page_title: 'Hetzner Cloud - Builders'
+sidebar_current: 'docs-builders-hetzner-cloud'
+---
+
+# Hetzner Cloud Builder
+
+Type: `hcloud`
+
+The `hcloud` Packer builder is able to create new images for use with
+[Hetzner Cloud](https://www.hetzner.cloud). The builder takes a source image,
+runs any provisioning necessary on the image after launching it, then snapshots
+it into a reusable image. This reusable image can then be used as the foundation
+of new servers that are launched within the Hetzner Cloud.
+
+The builder does *not* manage images. Once it creates an image, it is up to you
+to use it or delete it.
+
+## Configuration Reference
+
+There are many configuration options available for the builder. They are
+segmented below into two categories: required and optional parameters. Within
+each category, the available configuration keys are alphabetized.
+
+In addition to the options listed here, a
+[communicator](/docs/templates/communicator.html) can be configured for this
+builder.
+
+### Required:
+
+-   `token` (string) - The client TOKEN to use to access your account. It
+    can also be specified via environment variable `HCLOUD_TOKEN`,
+    if set.
+
+-   `image` (string) - ID or name of image to create server in. 
```

Comment:
This line doesn't make a ton of sense to me. Maybe "ID or name of image to launch server from." ?